apps/storybook/addons/background/src/preset/preview.js

```diff
@@ -1,18 +1,23 @@
-import { withOrigamiBackground } from '../withOrigamiBackground';
+import {withOrigamiBackground} from "../withOrigamiBackground"
 
-export const decorators = [withOrigamiBackground];
+export const decorators = [withOrigamiBackground]
 
 export const globalTypes = {
-    origamiBackground: {
-      name: 'Origami Background',
-      description: 'Background colour to show components on.',
-      toolbar: {
-        icon: 'photo',
-        items: [{
-          value: '',
-          title: 'recommended',
-        }, 'paper', 'wheat', 'slate'],
-        showName: false,
-      },
-    },
-  };
+	origamiBackground: {
+		name: "Origami Background",
+		description: "Background colour to show components on.",
+		toolbar: {
+			icon: "photo",
+			items: [
+				{
+					value: "",
+					title: "recommended",
+				},
+				"paper",
```

Comment:
Nice 👍 let's add white as an option!
```suggestion
				"paper",
				"white",
```